backend/app.py

```diff
@@ -6,7 +6,10 @@ class App:
     app = None
 
     def __init__(self):
+        from utils.eeljokes import db
```

Comment:
ditto: https://github.com/Emseers/Eelbot-ui/pull/1/files#r528263273

Move imports to the top of the file

Reply:
Done.